samples/nrf9160/fmfu_smp_svr/src/main.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <mgmt/fmfu_mgmt.h>
+#include <mgmt/fmfu_mgmt_stat.h>
+#include <modem/modem_info.h>
+#include <modem/nrf_modem_lib.h>
+#include <modem/at_cmd.h>
+
+void main(void)
+{
+	int err = nrf_modem_lib_init(NORMAL_MODE);
+
+	if (err == 0) {
+		char modem_version[MODEM_INFO_MAX_RESPONSE_SIZE];
+
+		at_cmd_init();
+		modem_info_init();
```

Comment:
Should we check the return value from these?

Reply:
it doesn't matter if they fail, but I can add it for correctness.

Reply:
kk, thanks!